### PR TITLE
fix: recognize if lambda is warm bug RD-1386

### PR DESCRIPTION
**Pull request by doriaviram-lumigo**

### CODE REVIEW

File: src/spans/awsSpan.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const getBasicSpan = () => {
   const account = awsAccountId;
 
   const readiness = isWarm() ? 'warm' : 'cold';
-  if (readiness === 'warm') {
+  if (readiness === 'cold') {
```
Comment (nirLumigo) [inline src/spans/awsSpan.js]: Don't we need to `setCold` in this case?

File: src/spans/awsSpan.test.js [hunk 2]
```diff
@@ -113,6 +113,12 @@ describe('awsSpan', () => {
     expect(awsSpan.getBasicSpan()).toEqual(expectedBasicSpan);
   });
 
+  test('getBasicSpan turn is warm', () => {
+    expect(utils.isWarm()).toEqual(false);
+    awsSpan.getBasicSpan();
```
Comment (nirLumigo) [inline src/spans/awsSpan.test.js]: Why `getBasicSpan()` should change it to be `'warm'`?

Comment (doriaviram-lumigo) [reply src/spans/awsSpan.test.js]: for the next time, you run the tracer, it was the behavior that Sagi wants, he just had a bug

File: src/spans/awsSpan.js (hunk 1 above)
Comment (doriaviram-lumigo) [reply src/spans/awsSpan.js]: if the lambda is cold, the next time you want its to be warm (not opposite)

**Pull request merged**